core/domain/skill_domain.py

```diff
@@ -585,6 +585,20 @@ def require_valid_skill_id(cls, skill_id):
         if len(skill_id) != 12:
             raise utils.ValidationError('Invalid skill id.')
 
+    @classmethod
+    def require_valid_skill_ids(cls, skill_ids):
+        """Checks whether the skill ids are valid.
+
+        Args:
+            skill_ids: list(str). The skill ids to validate.
+        """
+        for skill_id in skill_ids:
```

Comment:
Do not duplicate logic. Use require_valid_skill_id().

Reply:
Oh yeah sorry.
Done